reconcile/gitlab_members.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import logging
+
+import utils.gql as gql
+from utils.config import get_config
+from utils.gitlab_api import GitLabApi
+
+GROUPS_QUERY = """
+{
+    groups: gitlabgroups_v1{
+        managedGroups
+    }
+}
+"""
+
+USERS_QUERY = """
+{
+    users: users_v1{
+        redhat_username
+        roles{
+            permissions{
+                ... on PermissionGitlabGroups_v1{
+                    name
+                    group
+                }
+            }
+        }
+    }
+}
+  """
+
+def get_gitlab_api():
+    config = get_config()
+    gitlab_config = config['gitlab']
+    server = gitlab_config['server']
+    token = gitlab_config['token']
+    return GitLabApi(server, token, ssl_verify=False)
+
+
+def get_groups():
+    gqlapi = gql.get_api()
+    groups = gqlapi.query(GROUPS_QUERY)['groups']
+    group_dict = {}
+    for g in groups[0]['managedGroups']:
```

Comment:
is this assuming we have only one gitlab instance?
i know this will probably never change, but i'd still prefer we support `n` gitlab instances.
wdyt?